ssh/package.json

```diff
@@ -27,7 +27,8 @@
     "src"
   ],
   "dependencies": {
-    "@theia/core": "latest"
+    "@theia/core": "latest",
+    "axios": "latest"
```

Comment:
Does this package have approved CQ?

Reply:
There is a Piggyback https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15800